packages/core-library/system/app/internal/blocks/Hub/Settings/SettingsManagement/steps/MaintenanceMode/validation.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Property of the NCLEX Power.
+ * Reuse as a whole or in part is prohibited without permission.
+ * Created by the Software Strategy & Development Division
+ */
+import * as yup from "yup";
+
+export const createEnvironmentSchema = (
+  environmentList: string[],
+  activeField: string | undefined | null
```

Comment:
would there be an issue if we use this?
```suggestion
  activeField?: string | null
```